STAR/draft-ietf-acme-star.md

```diff
@@ -519,36 +553,37 @@ In order to enable the name delegation workflow defined in
 {{I-D.ietf-acme-star-delegation}} as well as to increase the reliability of the
 STAR ecosystem (see {{dependability}} for details), this document defines a
 mechanism that allows a server to advertise support for accessing
-star-certificate resources via unauthenticated GET (instead of, or in addition
-to, POST-as-GET), and a client to enable this service with per-Order
+star-certificate resources via unauthenticated GET (in addition
+to POST-as-GET), and a client to enable this service with per-Order
```

Comment:
Hmmm, this change has some implications. Maybe add: Even when unauthenticated GET has been negotiated, the server MUST also allow POST-as-GET requests to these resources.

Reply:
mmm why?  The client has clearly stated it wants to access those via GET.

Reply:
First, because this is what "in addition to" means to me... Also, pragmatically, you may have unauthenticated clients (the CDN endpoints) as well as authenticated ones (e.g. CLI tools used for debugging).

Reply:
OK, I see your point.